db/migrate/20210824160127_update_notified_at_with_created_at.rb

```diff
@@ -0,0 +1,13 @@
+class UpdateNotifiedAtWithCreatedAt < ActiveRecord::Migration[6.1]
+  def change
+    reversible do |dir|
+      dir.up do
+        PlanningApplication.all.map do |planning_application|
```

Comment:
also I believe it's good practice to redeclare your model class and its association if you're going to use it/them in your migration:

```ruby
class ValidationRequest < ApplicationRecord
  belongs_to :planning_application:
end

class PlanningApplication < ApplicationRecord
  has_many :validation_requests
end

end
```

Reply:
it doesn't have many validation_requests though, so instead of that association its a scope method, within PlanningApplication line 274:

```
  def validation_requests
    (description_change_validation_requests + replacement_document_validation_requests + additional_document_validation_requests + other_change_validation_requests + red_line_boundary_change_validation_requests).sort_by(&:created_at).reverse
  end
```

Reply:
I'll declare the PlanningApplication class atop though and use it with find_each instead